src/Org.Quickstart.API/Startup.cs

```diff
@@ -106,13 +106,20 @@ public void ConfigureServices(IServiceCollection services)
 	            //setup the database once everything is setup and running integration tests need to make sure database is fully working before running,hence running Synchronously
 	            appLifetime.ApplicationStarted.Register(() => {
 		            var db = app.ApplicationServices.GetService<DatabaseService>();
-		            db.SetupDatabase().RunSynchronously();
+		            db.CreateBucketCollection().RunSynchronously();
+		            db.CreateIndex().RunSynchronously();
 	            });
 		    } else {
 	            //setup the database once everything is setup and running
 	            appLifetime.ApplicationStarted.Register(async () => {
-		            var db = app.ApplicationServices.GetService<DatabaseService>();
-		            await db.SetupDatabase();
+		            var db = app.ApplicationServices.GetService<DatabaseService>();
+
+                    /* *** WARNING:  if you are using Capella, YOU MUST COMMMENT OUT THE NEXT LINE 
+                    Capellta users must create the bucket and collection before running this */
+                    await db.CreateBucketCollection();
```

Comment:
I think the approach here is incorrect, the collection must be created in both scenarios:

- The user must provide the database hostname/bucket/username/password for both capella and couchbase
- We should still create the collection and indexes automatically (capella allows us to do that)

Reply:
Sounds good to me - see the latest update as separated the three things:

-CreateBucket
-CreateCollection
-CreateIndex

Need to keep create bucket for unit tests so they can auto-create as needed.  The setup only runs create collection and indexes now and the documentation update shows them where to find how to create a bucket and user using the documentation.